ga4gh/cli.py

```diff
@@ -1400,9 +1400,9 @@ def getRepoParser():
     return parser
 
 
-def repo_main():
+def repo_main(args=None):
     parser = getRepoParser()
-    args = parser.parse_args()
+    args = parser.parse_args(args)
```

Comment:
It's confusing to reuse a function parameter as a local variable; can you rename the second 'args' to something please? (parsedArgs, eg.)